boot/boot_test.go

```diff
@@ -434,6 +437,10 @@ func (s *bootSetSuite) TestCoreParticipant20SetNextSameKernelSnap(c *C) {
 	// there was no attempt to enable a kernel
 	_, enableKernelCalls := s.bootloader.GetRunKernelImageFunctionSnapCalls("EnableTryKernel")
 	c.Assert(enableKernelCalls, Equals, 0)
+
+	// finally we didn't call SetBootVars on the bootloader because nothing
+	// changed
+	c.Assert(nSetBootVarCalls, Equals, s.bootloader.SetBootVarsCalls)
```

Comment:
(nitpick) silly question, could we simply do `c.Assert(s.bootloader.SetBootVarsCalls, Equals, 0)` here if nothing changed? It seems to work when I change this locally.

Reply:
added